Lib/json/__init__.py

```diff
@@ -342,6 +342,15 @@ def loads(s, *, encoding=None, cls=None, object_hook=None, parse_float=None,
                             f'not {s.__class__.__name__}')
         s = s.decode(detect_encoding(s), 'surrogatepass')
 
+    if "encoding" is not kw:
+        import warnings
+        warnings.warn(
+            "passing 'encoding' as a keyword argument is deprecated since"
+            " Python 3.1, is ignored and will be removed in Python 3.9.",
```

Comment:
"as a keyword argument" looks confusing.  ("Does it supported as positional argument?")

"'encoding' is ignored and deprecated.  It will be removed in Python 3.9".

Additionally, `del kw['encoding']` is required.  Otherwise, `JSONEncoder(..., **kw)` will fail.